### PR TITLE
Add the GraphQL Learn Content

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the GraphQL Learn content.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by RolandHewage — APPROVED** [review]

File: _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+                                 <div class="col-xs-11 col-md-11 col-lg-11 cNoPadding"> <h1>{{ page.title }}</h1></div>
+                                 <div class="col-xs-1 col-md-1 col-lg-1 cNoPadding cGithubContainer"><a class="icon icon-github" target="_blank" href="https://www.github.com/{{ site.github_username }}/{{ site.git_repo }}/blob/master/{{ page.path | replace: ".html", ".md"}}"></a></div>
+                                 <div class="col-xs-12 col-md-12 col-lg-12 cNoPadding">
+                                    <p>{{ page.intro }}</p>
+                                    <div class="cBallerinaTocContainer" id="table-of-content">
+                                       {% include toc.html html=content sanitize=true id="tree" %}
+                                    </div>
+                                    {{ content }}
+                                 </div>
```
Comment (anupama-pathirage) [inline _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html:44]: Unnecessary new lines

File: _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                  var link = $(this).attr('id');
+              
+                  var permLink = " <a class=\"permalink\" href=\"#"+link+"\"></a>";
+                  if(link){
+                     $(this).addClass('permalinkHeader').prepend(permLink) 
+                  } 
+                });
+                $('#tree').abixTreeList();  
+         });
+
```
Comment (anupama-pathirage) [inline _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html:93]: Unnecessary new lines

File: _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        var bodyclass = document.body;
+
+        if ($(this).scrollTop() > 200) {
+            $('.cTopLink').removeClass('dispalaynone');
+          
+        } else {
+            $('.cTopLink').addClass('dispalaynone');
+          
+        }
+
```
Comment (anupama-pathirage) [inline _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html:108]: Unnecessary new lines

File: learn.md [hunk 4]
```diff
@@ -269,6 +269,19 @@ redirect_from:
...
 </div>
 
+<div class="row">
+<div class="col-lg-6 col-md-6 col-sm-12 card"  style="margin-right:0px !important;">
+ <a href="/learn/ballerina-graphql-support/">
+    <h3 id="ballerina-graphql-support">Ballerina GraphQL support </h3></a>
+    <p >Details of all the features of the Ballerina GraphQL tools. </p>
+</div>
+<div class="col-lg-6 col-md-6 col-sm-12 card"  style="margin-right:0px !important;">
+ <!--<a href="/learn/ballerina-openapi-support/">
```
Comment (anupama-pathirage) [inline learn.md:279]: Remove commented blocks

File: _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html (hunk 2 above)
Comment (praneesha) [reply _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html:93]: Fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/4423/files.

File: _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html (hunk 3 above)
Comment (praneesha) [reply _layouts/ballerina-graphql-support-left-nav-pages-swanlake.html:108]: Fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/4423/files/.

File: learn.md (hunk 4 above)
Comment (praneesha) [reply learn.md:279]: This is updated now.

**Pull request merged**